Makefile

```diff
@@ -76,8 +79,14 @@ PUSH ?= false
 
 .PHONY: images
 images: $(KO)
-	KO_DOCKER_REPO=$(REPO)/$(EXTENSION_PREFIX)-$(NAME) $(KO) build --image-label org.opencontainers.image.source="https://github.com/stackitcloud/gardener-extension-acl" --sbom none -t $(TAG) --bare --platform linux/amd64,linux/arm64 --push=$(PUSH) ./cmd/gardener-extension-acl
-	KO_DOCKER_REPO=$(REPO)/$(EXTENSION_PREFIX)-$(ADMISSION_NAME) $(KO) build --image-label org.opencontainers.image.source="https://github.com/stackitcloud/gardener-extension-acl" --sbom none -t $(TAG) --bare --platform linux/amd64,linux/arm64 --push=$(PUSH) ./cmd/gardener-extension-admission-acl
+	KO_DOCKER_REPO=$(EXTENSION_IMAGE) $(KO) build --image-label org.opencontainers.image.source="https://github.com/stackitcloud/gardener-extension-acl" --sbom none -t $(TAG) --bare --platform linux/amd64,linux/arm64 --push=$(PUSH) ./cmd/gardener-extension-acl
```

Comment:
I don't like that the arguments for image build and chart pushing are repeated multiple times. Hence, I refactored the image build and chart push targets to be similar to something we already use internally.

I pushed the refactoring as a new commit to this PR. Please check if you like my proposal :)

Reply:
/lgtm
Thanks 😄